packages/core/client/src/appInfo/CurrentAppInfoProvider.tsx

```diff
@@ -22,6 +22,7 @@ export const useCurrentAppInfo = () => {
       };
       lang: string;
       version: string;
+      exportLimit?: number;
```

Comment:
1000000.00

Reply:
This will cause the entire application to hang and not be able to handle other requests